fli-guide-backend/seeders/20250113000000-seed-all-recipe-data.js

```diff
@@ -5,65 +5,28 @@ const path = require("path");
 
 /** @type {import('sequelize-cli').Seeder} */
 module.exports = {
-   async up(queryInterface) {
-      const sqlBaseDir = path.resolve(__dirname, "raw_sql");
+async up(queryInterface) {
+   const sqlBaseDir = path.resolve(__dirname, "raw_sql");
+   const transaction = await queryInterface.sequelize.transaction();
```

Comment:
We should run these in a transaction so either all these operations succeed or we rollback to our initial state